io.openliberty.microprofile.config.2.0.internal_fat_tck fails to run locally

```
io.openliberty.microprofile.config.internal_fat_tck.FATSuite
io.openliberty.microprofile.config.internal_fat_tck.FATSuite
launchConfig20Tck_EE8_FEATURES(io.openliberty.microprofile.config.internal_fat_tck.Config20TCKLauncher)
junit.framework.AssertionFailedError: 2020-11-11-08:32:53:926 No TCK test JUnit result files were found in the results directory which suggests the TCK tests did not run
Errors found in mvnOutput were:
### maven test output:
[ERROR] Failed to execute goal on project tck.runner.tck: Could not resolve dependencies for project io.openliberty.microprofile.config20.internal:tck.runner.tck:jar:2.0: Could not find artifact org.eclipse.microprofile.config:microprofile-config-tck:jar:2.0-ibm20201020 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

	at componenttest.topology.utils.MvnUtils.findJunitResultFiles(MvnUtils.java:182)
	at componenttest.topology.utils.MvnUtils.postProcessTestResults(MvnUtils.java:481)
	at componenttest.topology.utils.MvnUtils.runCleanTest(MvnUtils.java:634)
	at componenttest.topology.utils.MvnUtils.runTCKMvnCmd(MvnUtils.java:143)
	at componenttest.topology.utils.MvnUtils.runTCKMvnCmd(MvnUtils.java:113)
	at io.openliberty.microprofile.config.internal_fat_tck.Config20TCKLauncher.launchConfig20Tck(Config20TCKLauncher.java:57)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:197)
	at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:115)
	at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:318)
	at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:171)

```
It appears that it expects `org.eclipse.microprofile.config:microprofile-config-tck:jar:2.0-ibm20201020` to come from maven central.

--- a/dev/io.openliberty.microprofile.config.2.0.internal_fat_tck/publish/tckRunner/tck/pom.xml
+++ b/dev/io.openliberty.microprofile.config.2.0.internal_fat_tck/publish/tckRunner/tck/pom.xml
@@ -43,7 +43,17 @@
                 <scope>import</scope>
             </dependency>
         </dependencies>
-    </dependencyManagement>
+    </dependencyManagement> 
+    
+	<repositories>
+		<!-- This TCK is currently using a custom built artifact- microprofile-config-tck:2.0-ibm20201020.
+		If Artifactory is unavailable, the following DHE repository will be searched for the artifact -->
+		<repository>
+			<name>IBM DHE Maven repository</name>
+			<id>DHE</id>
+			<url>http://public.dhe.ibm.com/ibmdl/export/pub/software/olrepo</url>
+		</repository>
+	</repositories>
 
     <dependencies>
 
@@ -58,7 +68,7 @@
         <dependency>
             <groupId>org.eclipse.microprofile.config</groupId>
             <artifactId>microprofile-config-tck</artifactId>
-            <version>2.0-ibm20201020</version> <!--2.0-SNAPSHOT-->
+            <version>2.0-ibm20201020</version> 
         </dependency>
 
         <dependency>

--- a/dev/io.openliberty.microprofile.config.2.0.internal_fat_tck/publish/tckRunner/tck/tck-suite.xml
+++ b/dev/io.openliberty.microprofile.config.2.0.internal_fat_tck/publish/tckRunner/tck/tck-suite.xml
@@ -11,43 +11,11 @@
 <suite name="microprofile-config-2.0-tck" verbose="2"
     configfailurepolicy="continue">
     <test name="tck-package-org.eclipse.microprofile.config20.tck">
-        <!-- <classes>
-            <class name="org.eclipse.microprofile.config.tck.PropertyExpressionsTest"/>
-        </classes>-->
         <method-selectors>
             <method-selector>
                 <script language="beanshell">
                      <![CDATA[
-                     !method.getName().equals("escape")
-                     !method.getName().equals("noExpressionComposed")
-                     !method.getName().equals("multipleExpansions")
-                     !method.getName().equals("infiniteExpansion")
-                     !method.getName().equals("composedExpressions")
-                     !method.getName().equals("defaultExpressionEmpty")
-                     !method.getName().equals("defaultExpressionComposedEmpty")
-                     !method.getName().equals("simpleExpression")
-                     !method.getName().equals("expressionMissing")
-                     !method.getName().equals("arrayEscapes")
-                     !method.getName().equals("defaultExpressionComposed")
-                     !method.getName().equals("noExpression")
-                     !method.getName().equals("escapeBraces")
-                     !method.getName().equals("multipleExpressions")
-                     !method.getName().equals("defaultExpression")
                      !method.getName().equals("testInjectedConfigSerializable")
-                     !method.getName().equals("testConfigProfileWithDev")
-                     !method.getName().equals("testConfigPropertiesDefaultOnBean")
-                     !method.getName().equals("testConfigPropertiesNoPrefixOnBean")
-                     !method.getName().equals("testConfigPropertiesWithPrefix")
-                     !method.getName().equals("testNoConfigPropertiesAnnotationInjection")
-                     !method.getName().equals("testConfigPropertiesWithoutPrefix")
-                     !method.getName().equals("testConfigPropertiesPlainInjection")
-                     !method.getName().equals("testConfigPropertiesNoPrefixOnBeanThenSupplyPrefix")
-                     !method.getName().equals("expression")
-                     !method.getName().equals("expressionNoDefault")
-                     !method.getName().equals("configValueInjection")
-                     !method.getName().equals("configValue")
-                     !method.getName().equals("configValueEmpty")
-                     !method.getName().equals("test")
                 ]]>
                 </script>    
             </method-selector>

